doc/user/content/sql/update.md

```diff
@@ -11,15 +11,15 @@ menu:
 ## Syntax
 
 ```mzsql
-UPDATE <table_name> [ AS <alias> ]
+UPDATE <table_name> [ AS <table_alias> ]
    SET <column_name> = <expression> [, <column2_name> = <expression2>, ...]
-[WHERE <condition> ];
+[WHERE <condition(s)> ];
 ```
 
-Option | Description
-------|-----
-**AS**  | Only permit references to _table_name_ as the specified _alias_.
-**WHERE** | Only update rows which evaluate to `true` for _condition_.
+Option                        | Description
+------------------------------|------------
+**AS** <table_alias>          | If specified, you can only reference the table using the alias.
```

Comment:
I'm confused by this option:

```sql
CREATE TABLE mt (f int);

SELECT * FROM mt;

UPDATE mt AS mt_2 SET f = 1;

SELECT * FROM mt; --works

SELECT * FROM mt_2; --doesn't work
```

Reply:
Ah, it's in case the table name is mentioned in the `UPDATE` statement itself. I find the phrasing in the PostgreSQL documentation clearer:

> A substitute name for the target table. When an alias is provided, it completely hides the actual name of the table. For example, given UPDATE foo AS f, the remainder of the UPDATE statement must refer to this table as f not foo.

Reply:
Will add clarifying that within the UPDATE statement, can only reference using the alias.